### PR TITLE
Nuggeted people can be dunked into deep fryers and turned into a sentient nugget.

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

Seems fitting with the quickfire vote

# Wiki Documentation

# Changelog

:cl:  
rscadd: If you remove all the limbs off of a human and dunk them in a deep fryer, they turn into a sentient nugget! Yummy! 
/:cl:

### CODE REVIEW

**Comment by NCRLouTenant** [comment]
As stated in the discord, said nuggets should remain sentient.

(Okay but hear me out, nugget nuggets become isolated sentient diseases once eaten that only effect the person who eats it and no one else. Am I making things needlessly complex? Absolutely!)

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 1]
```diff
@@ -183,6 +183,21 @@ God bless America.
...
+			icon_state = "fryer_on"
+			
+			new /obj/item/reagent_containers/food/snacks/nugget(drop_location(src))
+			qdel(the_guy)
+			return
+		else
+			to_chat(user, span_warning("They still have limbs!"))
+			return
 				
 	if(user.pulling && user.a_intent == INTENT_GRAB && iscarbon(user.pulling) && reagents.total_volume && isliving(user.pulling))
```
Comment (SomeguyManperson) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm:204]: fun fact this exists

**Comment by KillerOrcaCora** [comment]
Finally another way to execute people in the kitchen !

**Comment by Readystorm546** [comment]
THis might be the best PR ever made

**Comment by ghost** [comment]
Based

**Review by ghost — APPROVED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 2]
```diff
@@ -183,6 +183,28 @@ God bless America.
...
+			icon_state = "fryer_on"
+			var /obj/item/reagent_containers/food/snacks/nugget/the_nugget = new /obj/item/reagent_containers/food/snacks/nugget(drop_location(src))
+			if(istype(the_guy) && the_guy.mind)
+				the_nugget.nugget_man = new(the_nugget)
+				the_nugget.nugget_man.real_name = the_nugget.name
+				the_nugget.nugget_man.name = the_nugget.name
+				the_nugget.nugget_man.stat = CONSCIOUS
+				the_guy.mind.transfer_to(the_nugget.nugget_man)
+			
+
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm]:
```suggestion
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm [hunk 3]
```diff
@@ -198,6 +218,7 @@ God bless America.
 		user.changeNext_move(CLICK_CD_MELEE)
 	return ..()
 
+
```
Comment (JamieD1) [inline code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm]:
```suggestion
```

**Review by JamieD1 — DISMISSED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**